<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipboardView</class>
 <widget class="QWidget" name="ClipboardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="styleSheet">
      <string notr="true">QListWidget::item {
    border: 1px solid lightgray;
    padding: 5px;
    margin: 2px;
}

QListWidget::item:selected {
    background-color: #a8c5f7;
    color: black;
}

QListWidget::item:hover {
    background-color: #e0e0e0;
}
</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
